.gitignore

```diff
@@ -1,2 +1,4 @@
 /vendor/
 composer.lock
+/.phpunit.result.cache
+/.idea
```

Comment:
I don't want this in .gitignore. You should have it in your own global gitignore :)

Reply:
Understandable. Thank you for the quick response :)